Projects/Windchill-App.md

```diff
@@ -0,0 +1,29 @@
+# Windchill
+
+Windchill combines the actual temperature with the wind speed to calculate
+the windchill factor, or what the perceived temperature is versus the actual
+temperature.
+
+## User Stories
+
+-   [ ] User can enter the actual temperature (Celcius) and the wind speed
+(kilometers per hour)
+-   [ ] User can press the `Calculate` button to display the wind chill
+-   [ ] User will receive and error message if data values are not entered
```

Comment:
Hmm.. "error message if ... `or if the resulting wind chill factor is greater`..."
Is this also an error message? Shouldn't this be the actual result and not an error? 🤔

Reply:
For more advanced developers this would actually be an assertion since by definition, wind chill factors must be less than the actual temperature. However, since this is a beginner app I didn't want to introduce `assert` so I left chose to specify it as an error. It really signifies there's a problem with the calculation if this case is true.

Reply:
Oh, I see... Thanks for the clarification!